### PR TITLE
General - Make use of `configOf`

### PR DESCRIPTION
**Pull request by jokoho48**
**When merged this pull request will:**
- Makes use of ConfigOf

### CODE REVIEW

File: addons/sys_core/fnc_findOcclusion.sqf [hunk 1]
```diff
@@ -74,12 +74,11 @@ private _intersectObjects = lineIntersectsObjs [_startPos, _endPos, _vehicleUnit
 
 
 {
-    private _typeString = typeOf _x;
-    if (_typeString != "") then {
+    if !(isNull _x) then {
```
Comment (jonpas) [inline addons/sys_core/fnc_findOcclusion.sqf]:
```suggestion
    if (!isNull _x) then {
```

**Review by jonpas — CHANGES_REQUESTED** [review]
Awesome!

**Review by jonpas — APPROVED** [review]

File: addons/sys_core/fnc_findOcclusion.sqf [hunk 2]
```diff
@@ -74,12 +74,11 @@ private _intersectObjects = lineIntersectsObjs [_startPos, _endPos, _vehicleUnit
 
 
 {
-    private _typeString = typeOf _x;
-    if (_typeString != "") then {
+    if (!isNull _x) then {
```
Comment (PabstMirror) [inline addons/sys_core/fnc_findOcclusion.sqf]: I think we want to revert this change
it's for detecting map objects that don't have a type

Comment (jokoho48) [reply addons/sys_core/fnc_findOcclusion.sqf]:
```suggestion
    if ((typeOf _x) != "") then {
```

**Review by PabstMirror — APPROVED** [review]
